src/test/java/core/basesyntax/converter/impl/DataConverterTest.java

```diff
@@ -0,0 +1,63 @@
+package core.basesyntax.converter.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class DataConverterTest {
+    private static DataConverter dataConverter;
+
+    @BeforeAll
+    static void beforeAll() {
+        dataConverter = new DataConverter();
+    }
+
+    @Test
+    void converter_fruitData_ShouldReturnWholeList() {
+        List<String> data = getData();
+        List<FruitTransaction> actual = dataConverter.convertToTransaction(data);
+        List<FruitTransaction> expected = List.of(
+                new FruitTransaction(FruitTransaction.Operation.BALANCE, "banana", 20),
+                new FruitTransaction(FruitTransaction.Operation.BALANCE, "apple", 100),
+                new FruitTransaction(FruitTransaction.Operation.PURCHASE, "banana", 13),
+                new FruitTransaction(FruitTransaction.Operation.RETURN, "apple", 10),
+                new FruitTransaction(FruitTransaction.Operation.PURCHASE, "apple", 20),
+                new FruitTransaction(FruitTransaction.Operation.PURCHASE, "banana", 5),
+                new FruitTransaction(FruitTransaction.Operation.SUPPLY, "banana", 50));
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    void convertToTransaction_invalidNumber_shouldThrowNumberFormatException() {
+        List<String> report = List.of(
+                "type,fruit,quantity",
+                "BUY,apple,invalidNumber");
+        assertThrows(RuntimeException.class, () -> dataConverter.convertToTransaction(report));
+    }
```

Comment:
If you want to keep this test, you should make some logic of handling for such situation, now this test makes no sense